### PR TITLE
Bug 1161970 - [Calendar] CalDAV is not correctly written

### PR DESCRIPTION
**Pull request by Hywan**
Note: Do we need to fix in other locales? Where are they?

(replace #29910)

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=74fe9bb0278e72238d0db71b513064caa398fbb8

File: build/test/resources/locales/zh-CN/apps/calendar/calendar.properties [hunk 1]
```diff
@@ -51,8 +51,8 @@ modify-account-unable-to-sync=无法同步
 modify-account-sign-in=登录
 
 # presets
-preset-caldav=CalDav
-preset-yahoo=Yahoo!
+preset-caldav=CalDAV
+Preset-yahoo=Yahoo!
```
Comment (KevinGrandon) [inline build/test/resources/locales/zh-CN/apps/calendar/calendar.properties]: Why this change?

File: apps/calendar/locales/calendar.en-US.properties [hunk 2]
```diff
@@ -49,7 +49,7 @@ modify-account-unable-to-sync=Unable to sync
 modify-account-sign-in=Sign in
 
 # presets
-preset-caldav=CalDav
+preset-caldav=CalDAV
```
Comment (KevinGrandon) [inline apps/calendar/locales/calendar.en-US.properties]: One gotcha of working with strings is that we need to notify the localization folks of string changes. We need to update the ```preset-caldav``` key in this file, and where it's used in order to do so.

File: build/test/resources/locales/zh-CN/apps/calendar/calendar.properties (hunk 1 above)
Comment (Hywan) [inline build/test/resources/locales/zh-CN/apps/calendar/calendar.properties]: An accident. Will revert it soon.

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=395a6f218db5bfb2c67de5824190090a608df850

File: apps/calendar/locales/calendar.en-US.properties (hunk 2 above)
Comment (Hywan) [inline apps/calendar/locales/calendar.en-US.properties]: All `preset-caldav` reference in the code has been updated with this PR.
